web_widget_image_download/static/src/js/web_widget_image_download.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2016 Flavio Corpa <flavio.corpa@tecnativa.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+ */
+openerp.web_widget_image_download = function(instance) {
+    'use strict';
+
+    instance.web.form.web_widget_image_download = instance.web.form.FieldBinaryImage.include({
+        render_value() {
```

Comment:
Quite [good](http://kangax.github.io/compat-table/es6/#test-object_literal_extensions), is the new *shorthand method* for object literals. 😄